### ISSUE TITLE
Set `process.env.NODE_ENV = 'production'` when bundling

### ISSUE THREAD

**Issue by SimenB** [asset-pipe/asset-pipe-build-server#78]
Currently we get the development version of e.g. react in production. Would be nice to avoid that

**Comment by trygve-lie** [comment]
Is there other ways to tell react to provide prod version than using `process.env.NODE_ENV`? vue.js will have the same issue.

**Comment by SimenB** [comment]
Docs says to use process.env.NODE_ENV

https://facebook.github.io/react/docs/optimizing-performance.html#use-the-production-build

**Comment by sveisvei** [comment]
I think it makes sense to inject NODE_ENV from the server process.env.

**Comment by SimenB** [comment]
Sorta related: we don't do minification

**Comment by SimenB** [comment]
@digitalsadhu this is done now, right?

**Comment by digitalsadhu** [comment]
Minification based on NODE_ENV=production yes. 
We didn't end up including envify yet though so still need to revisit that.

**Comment by SimenB** [comment]
ah, of course.

**Comment by digitalsadhu** [comment]
We now support minification. Defaults to process.env.NODE_ENV with fallback to 'development' if not defined. Latest client also allows overriding this on any given client call.

**Comment by theneva** [comment]
market-frontpage on Podium v2 says this:

![image](https://user-images.githubusercontent.com/1404650/42226698-eb95541e-7edf-11e8-8513-a4d71e40aff0.png)

Looks like we still have a ways to to here

**Comment by digitalsadhu** [comment]
Wonder what’s up here. 
Envify in use here:
https://github.com/asset-pipe/asset-pipe-js-reader/blob/742677c33284069c60f243334337d57a836417d6/lib/reader.js#L74

And I’ve seen this work with react.

**Comment by theneva** [comment]
Yep, I went digging and couldn't find anything obvious. Will have to continue looking into it tomorrow.
